### PR TITLE
The ability to bring up a full dev stack which will automatically tear itself down.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
There is one TODO - the actual ARN for the role that needs to bring up the stack needs to change, or else the perms for that role need to be changed.  Waiting on John for that.  So I figured I'd send this for review and then edit that before submitting.

Let's see if that testing github action works!  (Will add reviewers once I see whether that needs changes)

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by ndmckinley** [comment]
Cool - I think that's right.

File: infra/ttl.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    cfnresponse.send(event, context, status, {})
+                else:
+                    ttl = event['ResourceProperties']['ttl']
+                    responseData = {}
+                    responseData['cron_exp'] = deletion_time(ttl)
+                    cfnresponse.send(event, context, cfnresponse.SUCCESS, responseData)
+            except Exception as e:
+                logging.error('Exception: %s' % e, exc_info=True)
+                status = cfnresponse.FAILED
+                cfnresponse.send(event, context, status, {}, None)
```
Comment (hmhsu) [inline infra/ttl.yaml:115]: **Question**: Based on reading this [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-lambda-function-code-cfnresponsemodule.html), the fifth argument is default to `None`. Curious why we are setting it here, but not in other `send` invocation.

File: infra/iam_for_stack_ttl.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# provided by AWS quickstart - but not available hosted on s3, so reproduced here from
```
Comment (hmhsu) [inline infra/iam_for_stack_ttl.yaml:1]: **Question**: This file is not referenced in the main `stack.yaml` or anywhere else. Is that expected? Would like to know how it works.

**Review by hmhsu — APPROVED** [review]

**Comment by ndmckinley** [comment]
closing because staging does what this does, better!

**Pull request closed without merging**